src/Microsoft.Identity.Web/WebAppAuthenticationBuilderExtensions.cs

```diff
@@ -162,5 +162,18 @@ public static class WebAppAuthenticationBuilderExtensions
             return builder;
         }
 
+        /// <summary>
+        /// Ensures that the authority is a v2.0 authority
+        /// </summary>
+        /// <param name="options">OpenIdConnect options read from the config file
+        /// or set by the developer, for which we want to ensure the authority
+        /// is a v2.0 authority</param>
+        internal static void EnsureAuthorityIsV2_0(OpenIdConnectOptions options)
```

Comment:
We should also decide on how the naming convention for the version in the name. AuthorityHelpers.Is**V2**Authority is one way, EnsureAuthorityIs**V2_0** is another.